README.md

```diff
@@ -113,13 +111,13 @@ Write your tests in the stubbed out `/transform/testTransform.js` file.
 
 ### toUrl
 
-If you need to do more complex transformations to generate a URL, rather than just simple variable substitutions, go to  /src/toUrl.js and implement your logic in the function which is already stubbed out in that file. The function must return a string. The function body has access to these variables:
+If you need to do more complex transformations to generate a URL, rather than just simple variable substitutions, go to  /connection/connection.js and implement your logic in the function which is already stubbed out in that file. The function must return a string. The function body has access to these variables:
```

Comment:
I think this section has to describe the process for defining the url whether it's using toUrl or basic substitutions.